src/m_menu.c

```diff
@@ -5830,8 +5830,9 @@ boolean M_Responder (event_t* ev)
 		// Don't bind movement and turning to mouse wheel. It needs to
 		// be impossible to input a one-frame of movement automatically
 		// in speedrunning.
-		if ((ch == MOUSE_BUTTON_WHEELUP || ch == MOUSE_BUTTON_WHEELDOWN) &&
-		    s_input >= input_forward && s_input <= input_straferight)
+		if ((ch == MOUSE_BUTTON_WHEELUP || ch == MOUSE_BUTTON_WHEELDOWN || 
+         ch == MOUSE_BUTTON_WHEELLEFT || ch == MOUSE_BUTTON_WHEELRIGHT) &&
```

Comment:
Indentation somehow got borked here and in the next line.

Reply:
Huh, that's strange... Formatting seems to be correct on my end, but the commit looks borked indeed.